.github/workflows/monitor-components.yml

```diff
@@ -51,6 +51,8 @@ jobs:
             feed: https://github.com/petervanderdoes/gitflow-avh/tags.atom
           - label: curl
             feed: https://github.com/curl/curl/tags.atom
+          - label: libgcrypt
+            feed: https://github.com/gpg/libgcrypt/releases.atom
```

Comment:
There are occasionally non-releases (like libgcrypt-1.9-base and post-nuke-of-trailing-ws), pre-releases (like libgcrypt-1.5.0-beta1 and libgcrypt-1.4.2rc2) as well as maintenance releases for old releases in there. We might want a filter on here as well.

Reply:
Good point.

Reply:
Does the [current version](https://github.com/git-for-windows/git/compare/03e506f3fe9961d1e846161679dc3de11798118b..75f0e21527048f60af1bc260f208497da7ca0681) look better to you?

Reply:
Yes. It'll still give us the maintenance releases for old release lines, but thinking about GnuPG 2.3, that might be a good thing.